receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -88,35 +89,33 @@ func newTransaction(ctx context.Context, jobsMap *JobsMap, useStartTimeMetric bo
 // ensure *transaction has implemented the storage.Appender interface
 var _ storage.Appender = (*transaction)(nil)
 
-// there's no document on the first return value, however, it's somehow used in AddFast. I assume this is like a
-// uniqKey kind of thing for storage like a database, so that the operation can be perform faster with this key.
-// however, in this case, return 0 like what the prometheus remote store does shall be enough
-func (tr *transaction) Add(l labels.Labels, t int64, v float64) (uint64, error) {
-	return 0, tr.AddFast(l, 0, t, v)
-}
-
-// returning an error from this method can cause the whole appending transaction to be aborted and fail
-func (tr *transaction) AddFast(ls labels.Labels, _ uint64, t int64, v float64) error {
+// always returns 0 to disable label caching
```

Comment:
See https://github.com/prometheus/prometheus/blob/v2.19.2/storage/interface.go#L134
Returning 0 should lead to `AddFast` never being used.